doc/reference/orm.rst

```diff
@@ -124,9 +153,117 @@ for partners and one for countries::
         country = partner.country_id        # first pass prefetches all partners
         countries.add(country.name)         # first pass prefetches all countries
 
-Set operations
+.. _reference/orm/models:
+.. _reference/orm/model:
+
+Models
+======
+
+.. autoclass:: odoo.models.BaseModel()
+
+    .. autoattribute:: _auto
+    .. autoattribute:: _register
+
+    .. autoattribute:: _name
+    .. autoattribute:: _description
+
+    .. autoattribute:: _inherit
+    .. autoattribute:: _inherits
+
+    .. autoattribute:: _rec_name
+    .. autoattribute:: _order
+
+    .. autoattribute:: _date_name
+    .. autoattribute:: _fold_name
+
+    .. autoattribute:: _translate
```

Comment:
the list of param seems arbitrary. What about sql_constraints? `_abstract`? `parent_store` et al? Parenting is much more important that `_fold_name` imho... or just include all the attributes here instead of choosing arbitrary ones

Reply:
Done, I just left _needaction out of it as I'm not sure its current explanation is clear :D.